packages/frontend/src/components/PageLayout/Search/useSearchAutocompleteDialogs.tsx

```diff
@@ -90,26 +94,26 @@ const createAutocomplete = (
       items: [
         getScopeItem(
           <span>
-            <mark>{searchValue}</mark> i innboks
+            <mark>{searchValue}</mark> {t('search.autocomplete.inInbox')}
           </span>,
-          'Ingen treff',
+          t('search.noHits'),
         ),
       ],
-      groups: { noHits: { title: 'Ingen treff' } },
+      groups: { noHits: { title: t('search.noHits') } },
     } as AutocompleteProps;
   }
 
   return {
     items: [
       getScopeItem(
         <span>
-          <mark>{searchValue}</mark> i innboks
+          <mark>{searchValue}</mark> {t('search.autocomplete.inInbox')}
         </span>,
-        `${searchResults.length} treff`,
+        `${searchResults.length} ${t('word.hits')}`,
```

Comment:
This does not difference between singular and plural of "hit".
What about merging search.noHits with word.hits into something like `search.hits.label` which takes a number, 0="no hits", 1 ="{#count} hit", 1 > = "{#count} hits"?